components/NavBar.tsx

```diff
@@ -46,24 +46,32 @@ const LoginText = () => {
           ml={{
             base: '8px',
           }}
-          onClick={() => {
-            signIn('discord', { callbackUrl: "/dashboard" })
-          }}>
-          Login
+          // onClick={() => {
+          //   signIn()
+          // }}
+        >
+          <a href={params.url}>Login</a>
         </Text>
       )}
     </HStack>
   )
 }
 
-const MobileDrawer: React.FC<{ session: Session | null }> = ({ session }) => {
+const MobileDrawer: React.FC<Session> = (session) => {
   const { isOpen, onOpen, onClose } = useDisclosure()
   const btnRef = useRef(null);
   const router = useRouter();
 
-  const name = session?.user?.name
-  const image = session?.user?.image
-  
+  let name = '';
+  let image = '';
+  const loginUrl = `${process.env.NEXT_PUBLIC_GOVERNATOR_API_ENDPOINT}/auth/login`;
```

Comment:
so.. user with navigate to API? will need to test this in vercel

Reply:
yes. Hope it works. You need to rename the env variable to make it public

Reply:
so yeah FE is just navigating to the API, which just is a JSON page. Not useable. Maybe something is wrong? https://governator-staging-h8cdb8ie3-governator.vercel.app/

Reply:
it's a redirect to the discord oauth, has to come from same site that is why we redirect to backnd first. This part works. What doesn't work is sending the session cookie cross site. Lets see if we can get BE and FE under the same domain, that should solve the cookie and also we could implement this part with an axios request probably.